modules/andes-core/client/src/main/java/org/wso2/andes/client/QueueSenderAdapter.java

```diff
@@ -219,8 +218,9 @@ private void checkQueuePreConditions(Queue queue) throws JMSException
                }
                else
                {
-                   throw new InvalidDestinationException("Queue: " + queue
-                       + " is not a valid destination (no bindings on server");
+                   throw new InvalidDestinationException("The queue : " + queue + " is not a valid destination "
+                           + "(no bindings on the server). \nThis could be occurred due to invalid queue name or, "
+                           + "due to another error.");
```

Comment:
I think we can omit the sentence "This could be occurred due to invalid queue name or,  due to another error." It does not add any additional information.